memo_places_mobile/android/app/build.gradle

```diff
@@ -46,7 +49,8 @@ android {
         // You can update the following values to match your application needs.
         // For more information, see: https://docs.flutter.dev/deployment/android#reviewing-the-gradle-build-configuration.
         minSdkVersion flutter.minSdkVersion
-        targetSdkVersion flutter.targetSdkVersion
+        minSdkVersion 21
```

Comment:
Miko przypomnij mi dlaczego to zmieniałeś? 
Wiem że gadaliśmy o tym ale wypadło mi z głowy sorry.

Reply:
Mapy googla tego wymagają, jest w dokumentacji, jak chcesz mogę ci podesłać.